README.md

```diff
@@ -68,6 +68,40 @@ Where:
 * `DOJO_URL` is the URL where DefectDojo is installed.
 * `REPORT_FILE` is the Semgrep report path
 
+### Utility to integrate the results from the findings API in DefectDojo
```

Comment:
I can't leave a comment on the other header, but I think we should actually restructure this to have a top-level header of something like "Integrate Semgrep findings in DefectDojo" and then have one section for the common steps (API token, product/engagement) followed by separate sections for each script called something like:

* Using CLI JSON results
* Using Semgrep Findings API results

would be my recommendation.